solutions/elasticsearch-solution-project/agent-builder/get-started.md

```diff
@@ -15,7 +15,7 @@ Learn how to get started by enabling the {{agent-builder}} features and begin ch
 ::::::{stepper}
 ::::{step} Set up an Elastic deployment
 
-If you don't already have an Elastic deployment, refer to [Select your deployment type](/solutions/search/get-started.md#choose-your-deployment-type).
+If you don't already have an Elastic deployment, refer to [Get started with search](/solutions/search/get-started.md).
```

Comment:
this requires specifically an elasticsearch one right? can we be more particular about the requirement?

Reply:
this is changing in 9.3 so big asterisk about AB whose docs will need be moved and will be available to all